.babelrc

```diff
@@ -1,26 +1,59 @@
 {
-  "presets": [["env", {
-    "targets": {
-      "node": 4,
-      "browsers": [
-        "ie 9",
-        "ios 9",
-        "last 2 chrome versions",
-        "last 2 edge versions",
-        "last 2 firefox versions",
+  "env": {
```

Comment:
I think the `env` option is broken conceptually in some ways, that's why there is still discussion to deprecate it in favor of the new `.babelrc.js` config coming in babel 7. That's why there is currently also no effort put into fixing all the broken cases with `env`.

A workaround for babel 6.x is to create a local preset and reference it in the babel config. That way you have full control about which plugin is activated in which environment by simply using javascript + environment variables.

https://github.com/babel/babel/issues/5276